cmd/tailscale/cli/file.go

```diff
@@ -286,22 +287,109 @@ func runCpTargets(ctx context.Context, args []string) error {
 	return nil
 }
 
+type onConflict string
```

Comment:
// onConflict is a flag.Value for the --conflict flag's three string options.

Reply:
done